examples/bri-3/.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "tabWidth": 2
```

Comment:
AFAIK, this is the default setting. Recommended setting is 4 or higher. Should we not not increase the vaalue here?

Reply:
you are right, default is 2 i will remove this, i put 2 because files currently in project all had that width
i am ok with any tab settings as long as everybody agrees, if there is no objection on PR i will just remove this and we will use default value 2

Reply:
You meant default value **4** i guess.

Reply:
no default is 2, i think leaving  2 explicitly is better, it is same result with this and without this